.github/workflows/unittest.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       runs-on: ubuntu-20.04
       strategy:
         matrix:
-          python: [2.7, 3.6, 3.7, 3.8, 3.9]
+          python: [3.6, 3.9, 3.11]
```

Comment:
Can you also pull in the other changes from #334?

Reply:
@wpoely86 sure, but what do they do?

Reply:
Fix deprecation warnings and use the newest version of all actions. And add python 3.10.

Reply:
I was mainly wondering what the "newest version of all actions" means and implies.

Reply:
You need to read the change notes for that :stuck_out_tongue: 

You currently have deprecation warnings. As seen in #334  it still works and does what was intended.